client/my-sites/marketing/ultimate-traffic-guide/index.js

```diff
@@ -0,0 +1,316 @@
+import Experiment, {
+	DefaultVariation,
+	LoadingVariations,
+	Variation,
+} from 'calypso/components/experiment';
+import { isTrafficGuide } from 'calypso/lib/products-values';
+import { WPCOM_TRAFFIC_GUIDE } from 'calypso/lib/products-values/constants';
+
+/**
+ * Style dependencies
+ */
+import './style.scss';
+
+export const hasTrafficGuidePurchase = ( purchases ) =>
+	!! ( purchases && purchases.find( ( purchase ) => isTrafficGuide( purchase ) ) );
+
+export const downloadTrafficGuide = () => {
+	window.location.href = 'https://public-api.wordpress.com/wpcom/v2/traffic-guide-download';
+};
+
+export default function UltimateTrafficGuide() {
+	const translate = useTranslate();
+	const userId = useSelector( ( state ) => getCurrentUserId( state ) );
+	const isLoading = useSelector( ( state ) => isFetchingUserPurchases( state ) );
+	const purchases = useSelector( ( state ) => getUserPurchases( state, userId ) );
+	const hasPurchase = hasTrafficGuidePurchase( purchases );
+
+	const renderContent = () => {
+		return (
+			<>
+				<PageViewTracker
+					path={ '/marketing/ultimate-traffic-guide' }
+					title={ 'Ultimate Traffic Guide' }
+					properties={ { has_purchase: hasPurchase } }
+				/>
+				{ hasPurchase ? (
+					<DownloadPage translate={ translate } />
+				) : (
+					<SalesPage translate={ translate } />
+				) }
+			</>
+		);
+	};
+
+	return (
+		<CompactCard className="ultimate-traffic-guide">
+			{ ! purchases && <QueryUserPurchases userId={ userId } /> }
+			{ isLoading || ! purchases ? <Placeholder num={ 5 } /> : renderContent() }
+		</CompactCard>
+	);
+}
+
+const Placeholder = ( { num = 1 } ) =>
+	[ ...Array( num ) ].map( ( e, i ) => (
+		<div key={ `${ i }` } className="ultimate-traffic-guide__placeholder">
+			<div className="ultimate-traffic-guide__placeholder-animation"></div>
+		</div>
+	) );
+
+const SalesPage = ( { translate } ) => {
+	const cost = useSelector( ( state ) => getProductCost( state, WPCOM_TRAFFIC_GUIDE ) );
+	const currencyCode = useSelector( ( state ) => getCurrentUserCurrencyCode( state ) );
+	const isLoading = useSelector( ( state ) => isProductsListFetching( state ) );
+	const siteId = useSelector( ( state ) => getSelectedSiteId( state ) );
+	const siteSlug = useSelector( ( state ) => getSiteSlug( state, siteId ) );
+
+	/**
+	 * The reference cost is calculated to the nearest 100
+	 */
+	const referenceCost = Math.floor( ( cost * 7 ) / 100 ) * 100;
+
+	const displayCost = formatCurrency( cost, currencyCode, { stripZeros: true } );
+	const displayReferenceCost = formatCurrency( referenceCost, currencyCode, { stripZeros: true } );
+
+	const cta = () => (
+		<p>
+			<Button href={ `/checkout/${ siteSlug }/${ WPCOM_TRAFFIC_GUIDE }` } primary>
+				{ translate( 'Click here to buy the guide now!' ) }
+			</Button>
+		</p>
+	);
+
+	const defaultVariation = () => (
+		<>
+			<h1 className="ultimate-traffic-guide__header">
+				{ translate( 'Introducing the WordPress.com Ultimate Traffic Guide' ) }
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **1** times:
`translate( 'Introducing The Ultimate Traffic Guide' )` *ES Score: 8*